### PR TITLE
Traversability

### PR DESCRIPTION
**Pull request by gknisely**
Driveability -----> Traversability

### CODE REVIEW

**Comment by dnesbitt61** [comment]
I don't like renaming local_driveability). The method name would need to be: local_auto_traversability which just seems long and confusing to me! I don't mind changing the enum name and explain in the comment it is based on the mode of travel.

**Comment by gknisely** [comment]
@dgearhart loves  local_auto_traversability i am sure

File: valhalla/baldr/nodeinfo.h [hunk 1]
```diff
@@ -221,7 +221,7 @@ class NodeInfo {
 
   // Node type
   struct NodeTypeInfo {
-    uint32_t local_driveability : 16; // Driveability for local edges (up to
+    uint32_t auto_traversability : 16; // Traversability for local edges (up to
```
Comment (dgearhart) [inline valhalla/baldr/nodeinfo.h]: should we keep the local prefix?

**Comment by dgearhart** [comment]
:shipit:

**Pull request merged**